spec/models/passenger_spec.rb

```diff
@@ -156,5 +156,22 @@
         expect(call).to eq(3.business_days.after(Time.zone.today))
       end
     end
+
+    describe '.to_csv' do
+      subject(:call) { described_class.all.to_csv }
+
+      before do
+        create(:passenger, name: 'John Doe', needs_longer_rides: true, permanent: true)
+        create(:passenger, name: 'Jane Smith', needs_longer_rides: false, permanent: false)
```

Comment:
```suggestion
        create(:passenger, name: 'John Doe', permanent: true)
        create(:passenger, name: 'Jane Smith', permanent: false)
```
Not passenger attributes

Reply:
```suggestion
        longer_rides_mobility_device = create(:mobility_device, needs_longer_rides: true)
        create(:passenger, name: 'John Doe', mobility_device: longer_rides_mobility_device, permanent: true)
        create(:passenger, name: 'Jane Smith', permanent: false)
```
Make one of the passengers need longer rides, by giving them a mobility device that needs longer rides